flink-runtime/src/test/java/org/apache/flink/runtime/zookeeper/ZooKeeperStateHandleStoreTest.java

```diff
@@ -79,24 +68,24 @@
  *   <li>Correct ordering of ZooKeeper and state handle operations
  * </ul>
  */
-public class ZooKeeperStateHandleStoreTest extends TestLogger {
+class ZooKeeperStateHandleStoreTest extends TestLogger {
```

Comment:
```suggestion
class ZooKeeperStateHandleStoreTest {
```

`TestLogger` is a JUnit4 component and should be removed as part of the migration.

Reply:
Hi @XComp , thanks for your review!

This is my mistake, I have updated all TestLogger to `TestLoggerExtension`.